standard-cluster-services/src/main/java/org/terracotta/entity/IEntityMessenger.java

```diff
@@ -39,6 +39,18 @@
   void messageSelf(EntityMessage message) throws MessageCodecException;
 
   /**
+   * Defer a message based on another message, which will be sent in the future.
```

Comment:
This hasn't cleared it up for me at all.

I don't understand the use case from EhCache or TCStore.

I don't understand what it even means to defer retirement. I thought messages are "retired" once they are delivered to all servers. How do you defer that?

Why is there a release() method if the message is always retired when the second one is sent?